src/layout/section/itemsSection/itemsSection.tsx

```diff
@@ -14,13 +15,14 @@ export type ItemsSectionProps = Readonly<{
  * Use with two items.
  */
 export const ItemsSection = (props: ItemsSectionProps) => {
-  const { className, children, tag = <div /> } = props
+  const { className, children, noHorizontalSpacing, tag = <div /> } = props
 
   return (
     <BaseSection
       tagName={tag.type}
       className={className}
       contentClassName="kirk-items-section-content"
+      noHorizontalSpacing={noHorizontalSpacing}
```

Comment:
I'm guessing `ItemSection` should only take `Item*` as children. So I think it's safe to hardcode noHorizontalSpacing to true instead of forwarding it.
This is because all `Item*`s have been normalized.

Reply:
Even better! Done you can check it again

Reply:
Sorry didn't see the notification :(